exonum/src/runtime/mod.rs

```diff
@@ -209,11 +209,11 @@ impl From<RuntimeIdentifier> for u32 {
 ///
 /// ```text
 /// LIFE ::= initialize (GENESIS | RESUME) BLOCK* shutdown
-/// GENESIS ::= (deploy_artifact | initiate_adding_service commit_service)* after_commit
-/// RESUME ::= (deploy_artifact | commit_service)* on_resume
-/// BLOCK* ::= PROPOSAL+ COMMIT
+/// GENESIS ::= (deploy_artifact | initiate_adding_service update_service_status)* after_commit
+/// RESUME ::= (deploy_artifact | update_service_status)* on_resume
+/// BLOCK* ::= PROPOSAL + COMMIT
```

Comment:
`+` is a quantity modifier ("one or more"). IMO, it makes little sense to separate it from the quantified action (i.e., `PROPOSAL`).

Reply:
Oh, I wasn't be attentive, I'll correct this.